documentation/book/proc-upgrading-the-cluster-operator-0-9-0-to-0-10-0.adoc

```diff
@@ -3,75 +3,170 @@
 // assembly-upgrade.adoc
 
 [id='proc-upgrading-the-cluster-operator-{context}']
-= Upgrading the Cluster Operator from {ProductVersionUpgradingFrom} to {ProductVersionUpgradingTo}
+= Upgrading the Cluster Operator from {ProductLongName} {ProductVersionUpgradingFrom} to {ProductVersionUpgradingTo}
 
 ifdef::StrimziUpgrades[]
-This procedure will describe how to upgrade a Cluster Operator deployment from version 0.9.x to version {ProductVersionUpgradingTo}. 
+This procedure describes how to upgrade a Cluster Operator deployment from version 0.9.x to version {ProductVersionUpgradingTo}.
```

Comment:
`proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc` is using hard-coded version numbers. Here we've got this attribute, but it has to have the value 0.10.0 for this procedure to be correct, AFAICs. If there's no value in having the attribute then we should remove it and just used hard-coded version numbers here too, right?

Reply:
Using these attributes, downstream the chapter becomes 'Upgrading the Cluster Operator from 1.0.0 to 1.1.0'. We only have the one upgrade path downstream too; as opposed to 'Upgrading the Cluster Operator from 0.10.0 to 0.11.0' and 'Upgrading the Cluster Operator from 0.9.0 to 0.10.0 upstream'.